content/docs/command-reference/add.md

```diff
@@ -158,8 +158,9 @@ outs:
     path: data.xml
 ```
 
-This is a standard `.dvc` file with only one output (`outs` field). The hash
-value (`md5` field) corresponds to a file path in the <abbr>cache</abbr>.
+This is a standard `.dvc` file with only one output (`outs` field). The `outs`
+field can have any hash type (`md5` field in this case). The hash value
+corresponds to a file path in the <abbr>cache</abbr>.
```

Comment:
Hmm... No. In this case it's `md5` specifically. Look at the YAML example above.